src/activities/quizzes/QuizEntity.js

```diff
@@ -257,6 +257,15 @@ export class QuizEntity extends Entity {
 		return showResultsOverviewEntity && showResultsOverviewEntity.hasClass(Classes.quizzes.checked);
 	}
 
+	suggestContent() {
+		if (!this.isStudySupportEnabledVisible()) {
```

Comment:
I've been meaning to ask, what does `isStudySupportEnabledVisible` correspond to? I know `isStudySupportEnabled` reflects the value of the toggle for LSS.

Reply:
`isStudySupportEnabledVisible` is if study support is even available to the user, i.e. it's "enabled" via the flag/config. `isStudySupportEnabled` means it's actually set to be on.

Reply:
Thanks!